### ISSUE TITLE
Feature Proposal—Accessibility Validation and Tools

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/ketty#841]
> **Migrated from GitLab**
> **Original Author:** Erich van Rijn (`@erichvanrijn`)
> **Original Link:** https://gitlab.coko.foundation/coko-org/products/ketty/ketty/-/issues/211

This is a placeholder feature proposal to develop tools for validating EPUB outputs against accessibility guidelines as codified by the W3C and other standards organizations.  This could include anythingn from validating the inclusion of alt-text and alerting users to missing alt-text for images/media to validating whole book files against api-driven accessibility validation services such as those being developed by the DAISY consortium.

**Comment by yannisbarlas** [comment]
> **Migrated Comment**
> **Author:** Adam Hyde (`adam`)
> **Date:** 2018-11-01T05:53:19.998Z

`erichvanrijn`  by 'placeholder' do you mean you will add more details later?

**Comment by yannisbarlas** [comment]
> **Migrated Comment**
> **Author:** Adam Hyde (`adam`)
> **Date:** 2018-11-01T22:31:14.158Z

`erichvanrijn`  `taftman`  `Alison`  - shall we perhaps for a workgroup around this to look at the issue and identify what we need to take care of? I'd love an clear simple doc with easy wins and harder issues outlined.... this woulf be absolutely amazing

**Comment by yannisbarlas** [comment]
> **Migrated Comment**
> **Author:** Jason Colman (`taftman`)
> **Date:** 2018-11-01T23:38:07.720Z

`adam`  +1 to the workgroup idea

**Comment by yannisbarlas** [comment]
> **Migrated Comment**
> **Author:** Adam Hyde (`adam`)
> **Date:** 2018-11-02T00:14:04.689Z

`Alison`  - can you facilitate this? maybe send an email to the list calling out `taftman`  and `erichvanrijn`  interest and see who else may wish to join? then maybe we need a kick off remote meet soonish

**Comment by yannisbarlas** [comment]
> **Migrated Comment**
> **Author:** Alison McGonagle-O’Connell (`Alison`)
> **Date:** 2018-11-02T00:32:29.906Z

Sure can, `adam` !

**Comment by yannisbarlas** [comment]
> **Migrated Comment**
> **Author:** Lisa Schiff (`lschiff`)
> **Date:** 2018-11-09T00:07:58.757Z

Underscoring Justin's comment.  Any tool we use must be accessible.
